src/bench/block_assemble.cpp

```diff
@@ -41,7 +41,8 @@ static CTxIn MineBlock(const CScript& coinbase_scriptPubKey)
     auto block = PrepareBlock(coinbase_scriptPubKey);
 
     while (!CheckProofOfWork(block->GetHash(), block->nBits, Params().GetConsensus())) {
-        assert(++block->nNonce);
+        auto nNonce = ++block->nNonce;
+        assert(nNonce);
```

Comment:
You can just remove the assert. This is regtest, so a wrap is "impossible" anyway.

Reply:
Like this?

```
     while (!CheckProofOfWork(block->GetHash(), block->nBits, Params().GetConsensus())) {
         ++block->nNonce;
     }
```

Reply:
sure